### PR TITLE
Feat: adds Table.toArray function

### PR DESCRIPTION
**Pull request by celsobonutti**
This PR adds a binding for the [Table.toArray](https://dexie.org/docs/Table/Table.toArray()) method.
This is quite useful for fetching the whole table items, which is something we are needing for a current project in which we are using `rescript-dexie`.

### CODE REVIEW

**Comment by dusty-phillips** [comment]
Thanks! Can you also add a demo/test to https://github.com/dusty-phillips/rescript-dexie/blob/main/tests/TestTable.test.res (the world's best example of why we need async/await in Rescript 🤢).

**Comment by celsobonutti** [comment]
Hahahahah for sure! I'll do it in a moment. 😆

**Comment by celsobonutti** [comment]
Done! 😃

File: tests/TestWhere.test.mjs [hunk 1]
```diff
@@ -211,7 +211,6 @@ Zora$1.test("Where clauses", (function (t) {
                                                       ]
                                                     ]).toArray();
                                       })), (function (items) {
-                                    console.log(items);
```
Comment (dusty-phillips) [inline tests/TestWhere.test.mjs:214]: bonus points for cleaning up my garbage. :-D

**Comment by dusty-phillips** [comment]
awesome, thanks! Just released 0.3.3 with this change.

**Pull request merged**